src/components/Offer.vue

```diff
@@ -0,0 +1,379 @@
+    </v-container>
+  </v-card>
+
+  <v-container class="px-10">
+    <v-row class="justify-end pb-15 mr-12 mt-3">
+      <v-col md="3">
+        <v-btn type="submit" block class="mt-2" variant="outlined" color="blue"
+          size="large" @click="canselDialog = true">
+          Cancel
+        </v-btn>
+        </v-col>
+      <v-col md="3">
+        <v-btn type="submit" block class="mt-2" variant="flat" color="blue"
+          size="large" @click="createOffer">
+          Send Offer
+        </v-btn>
+      </v-col>
+    </v-row>
+  </v-container>
+
+  <div class="text-center">
+    <v-dialog v-model="canselDialog" activator="parent" width="500">
+      <v-card>
+        <v-toolbar color="primary" title="CANCELLATION CONFIRMATION" height="50"></v-toolbar>
+          <v-card-text class="text-center">
+            <div>Do you really want to cancel the Offer creation?</div>
+            <div>All you entered data will be lost</div>
+          </v-card-text>
+          <v-row class="justify-center mb-8 mt-3">
+            <v-btn @click="canselDialog = false" width="160" class="mx-2">No</v-btn>
+            <v-btn href="/module/bidder/tenders" color="primary" width="160" class="mx-2">Yes</v-btn>
+          </v-row>
+      </v-card>
+    </v-dialog>
+  </div>
+
+</template>
+
+<script>
+import { restApiConfig } from "@/rest.api.config"
+
+export default {
+  data: () => ({
+    countries: [],
+    currencies: [],
+    country: null,
+    currency: null,
+    isDisabled: true,
+    offer: {
+      tenderId: '',
+      organizationName: '',
+      nationalRegistrationNumber: '',
+      countryId: '',
+      city: '',
+      firstName: '',
+      lastName: '',
+      phone: '',
+      bidPrice: 0,
+      currencyId: '',
+      documentName: '',
+    },
+    valid: false,
+    dialog: false,
+    canselDialog: false,
+    isDocument: false,
+    document: null,
+    documentUrl: '',
+  }),
+
+  methods: {
+    onFileChangedDocument(event) {
+      this.isDocument = true;
+      this.document = event.target.files[0];
+    },
+
+    openDialog(document) {
+      this.documentUrl = URL.createObjectURL(document);
+      this.dialog = true;
+    },
+
+    getCountries() {
+      fetch(`${restApiConfig.host}${restApiConfig.countries}`, {
+        method: 'GET',
+        credentials: 'include',
+        headers: {
+          'Accept': 'application/json',
+        }
+      })
+        .then(response => response.json())
+        .then(dataFromResopnse => this.countries = dataFromResopnse)
+    },
+
+    getCurrencies() {
+      fetch(`${restApiConfig.host}${restApiConfig.currencies}`, {
+        method: 'GET',
+        credentials: 'include',
+        headers: {
+          'Accept': 'application/json',
+        },
+      })
+        .then(response => response.json())
+        .then(dataFromResopnse => this.currencies = dataFromResopnse);
+    },
+
+    async createOffer() {
+      this.$router.push("/module/bidder/tenders")
+      Promise.all([
```

Comment:
you should add _await_ before Promise.all and remove the await keyword on the next line